.build/Dockerfile

```diff
@@ -6,7 +6,7 @@
 # BUILD_DIR  = .build
```

Comment:
in general committing a hidden directory seems weird to me. also committing things that could be generated by a ci is not a good idea, unless dockerhub needs this folder for some reason.